examples/md-flexible/src/BoundaryConditions.h

```diff
@@ -104,12 +104,13 @@ std::vector<Particle> identifyNewHaloParticles(autopas::AutoPas<Particle> &autoP
           }
         }
         // Find non-halo particles in the designated region, create periodic copies of them and insert those as halos.
-        for (auto iter = autoPas.getRegionIterator(min, max, autopas::IteratorBehavior::owned); iter.isValid();
-             ++iter) {
-          auto particleCopy = *iter;
+
+        auto forEachInRegionLambda = [&](Particle particle) {
```

Comment:
```suggestion
        auto forEachInRegionLambda = [&](const Particle& particle) {
```
here, you copy particles twice!